packages/protocol/migrations/25_stableToken_registry.ts

```diff
@@ -0,0 +1,16 @@
+/* tslint:disable:no-console */
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import { deploymentForCoreContract } from '@celo/protocol/lib/web3-utils'
+import { config } from '@celo/protocol/migrationsConfig'
+import { StableTokenRegistryInstance } from 'types'
+
+const initializeArgs = async (): Promise<any[]> => {
+  return [config.stableTokenRegistry.fiatTicker, config.stableTokenRegistry.stableTokenContractName]
```

Comment:
This could be removed, but I'd leave a comment about updating this when we have a new stable